lib/vector-core/src/metrics/mod.rs

```diff
@@ -109,25 +109,25 @@ pub fn get_controller() -> crate::Result<&'static Controller> {
 
 /// Take a snapshot of all gathered metrics and expose them as metric
 /// [`Event`]s.
-pub fn capture_metrics(controller: &Controller) -> impl Iterator<Item = Event> {
-    let mut events: Vec<Event> = Vec::new();
+pub fn capture_metrics(controller: &Controller) -> impl Iterator<Item = Metric> {
+    let mut metrics: Vec<Metric> = Vec::new();
     controller.registry.visit(|_kind, (key, handle)| {
-        events.push(Metric::from_metric_kv(key, handle.get_inner()).into());
+        metrics.push(Metric::from_metric_kv(key, handle.get_inner()));
     });
 
-    // Add alias `events_processed_total` for `events_out_total`.
-    for i in 0..events.len() {
-        let metric = events[i].as_metric();
-        if metric.name() == "events_out_total" {
-            let alias = metric.clone().with_name("processed_events_total");
-            events.push(alias.into());
+    // Add alias `metrics_processed_total` for `metrics_out_total`.
```

Comment:
Was this change (`events_out_total` vs `metrics_out_total`) intentional?

Reply:
Ugh, nope, that was an aggressive find/replace. Thanks for pointing that out!